doc/sphinx/analysis.rst

```diff
@@ -710,6 +710,8 @@ The following observables are available:
      :math:`\frac{1}{\sum_i m_i} \left( \sum_i m_i v^x_i, \sum_i m_i v^y_i, \sum_i m_i v^z_i\right)`
    - ComForce: Sum of the forces on the particles
      :math:`\sum_i f^x_i, \sum_i f^y_i, \sum_i f^z_i`
+   - ChainAngles: Angles between bonds. The `i`-th value in the result vector corresponds to the angle between
+     bonds that are separated by `i` bonds.
```

Comment:
Wouldn't a name `PersistenceAngles` or something similar help differentiate this observable from `ParticleAngles`, which also calculates chain angles but without average?